opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -26,15 +26,19 @@
 
 
 """
+import enum
 from abc import ABC, abstractmethod
-from typing import Dict, List, Tuple, Type, Union
+from typing import Callable, Optional, Tuple, Type, Union
 
-from opentelemetry.metrics.handle import (
-    CounterHandle,
-    GaugeHandle,
-    MeasureHandle,
-)
-from opentelemetry.trace import SpanContext
+from opentelemetry.util import loader
+
+ValueType = Union[int, float]
+
+
+class MetricKind(enum.Enum):
```

Comment:
We don't need that enum. Just use the types defined in the mapping directly every time you would use the corresponding enums (in Python, types are first-class objects after all).

Reply:
I use the map to map type -> class directly so I can just call the variable as a constructor in `create_metric`. 
``
        return METRIC_KIND_MAP[metric_type](
            name,
            description,
            unit,
            value_type,
            label_keys=label_keys,
            enabled=enabled,
            monotonic=monotonic,
        )
``

Reply:
If you pass the type direclty in a parameter `metric_type` (of type `type` aka `Type[SomeConcreteMetricType]` you can do that too, simply call `metric_type(name, description, ...)`.

Reply:
That is awesome. Will do that.